src/explore-education-statistics-admin/src/pages/release/components/PreReleaseUserAccessForm.tsx

```diff
@@ -86,7 +86,13 @@ const PreReleaseUserAccessForm = ({
           validationSchema={Yup.object<FormValues>({
             email: Yup.string()
               .required('Enter an email address')
-              .email('Enter a valid email address'),
+              .email('Enter a valid @education.gov.uk email address')
+              .test({
+                name: 'email format',
+                message: 'Enter a valid @education.gov.uk email address',
+                test: (value: string) =>
+                  value.split('@', 2)[1] === 'education.gov.uk',
```

Comment:
This'll probably incorrectly allow something like `'test@education.gov.uk@test'`. The suggestion to limit it was probably the wrong choice of words. I actually meant split and check that it has only 2 parts before checking the second part.

Reply:
@ntsim It would still throw the error because 'education.gov.uk@test' wouldn't equal 'education.gov.uk'

Reply:
What's the problem? That would be the expected behaviour if it was validating something like `test@education.gov.uk@test`.

Reply:
Doesn't matter I read the comment again and it made more sense